CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## 0.22.0
 
+* Add `v1beta2` version for all resources. `v1beta2` removes all deprecated fields.
```

Comment:
Maybe best to specify exactly where the new version is added. Do you mean added to the documentation and Schema Reference?

Reply:
It is overall change for everything.